### PR TITLE
feat: create navbar and footer

### PR DESCRIPTION
**Pull request by andikadeniardi**

### CODE REVIEW

File: app/Components/Footer/index.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+const Footer = () => {
+  return (
+    <div className="bg-gray-800 text-white py-4">
+      <div className="max-w-7xl mx-auto px-4">
+        <div className="flex flex-col sm:flex-row justify-between items-center">
+          <div className="mb-2 sm:mb-0">
+            <h2 className="text-lg font-bold">FilixFlop </h2>
```
Comment (zenzett) [inline app/Components/Footer/index.tsx:9]: typo nih, ganti jadi FlixFlop

File: app/Components/Navbar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+const Navbar = () => {
+  return (
+    <div className="bg-gray-800">
+      <div className="max-w-7xl mx-auto px-2 sm:px-6 lg:px-8">
+        <div className="relative flex items-center justify-between h-20">
+          <div className="flex-1 flex items-center justify-center sm:items-stretch sm:justify-start">
+            <div className="flex-shrink-0">
+              <h1 className="text-white text-lg font-bold">FilixFlop</h1>
```
Comment (zenzett) [inline app/Components/Navbar/index.tsx:10]: typo juga nih

File: app/Components/Footer/index.tsx (hunk 1 above)
Comment (andikadeniardi) [reply app/Components/Footer/index.tsx:9]: done

File: app/Components/Navbar/index.tsx (hunk 2 above)
Comment (andikadeniardi) [reply app/Components/Navbar/index.tsx:10]: done

**Pull request closed without merging**